plugins/spa/drng_livefeed.go

```diff
@@ -0,0 +1,57 @@
+package spa
+
+import (
+	"encoding/hex"
+	"time"
+
+	"github.com/iotaledger/hive.go/daemon"
+	"github.com/iotaledger/hive.go/events"
+	"github.com/iotaledger/hive.go/workerpool"
+
+	"github.com/iotaledger/goshimmer/packages/binary/drng/state"
+	"github.com/iotaledger/goshimmer/packages/shutdown"
+	"github.com/iotaledger/goshimmer/plugins/drng"
+)
+
+var drngLiveFeedWorkerCount = 1
+var drngLiveFeedWorkerQueueSize = 50
+var drngLiveFeedWorkerPool *workerpool.WorkerPool
+
+func configureDrngLiveFeed() {
+	drngLiveFeedWorkerPool = workerpool.New(func(task workerpool.Task) {
+		newRandomness := task.Param(0).(state.Randomness)
+		log.Info(newRandomness)
```

Comment:
We should probably log this somewhere in the DRNG plugin instead of doing it here.